src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -72,8 +73,15 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Get
                 If m_ControlData Is Nothing Then
 
+                    Dim constantTargetFramework As String
```

Comment:
I wouldn't use "constant" in the name here; maybe "targetFrameworkPropertyName" instead?

Reply:
Makes sense, thanks!